src/iden3comm/handlers/contract-request.ts

```diff
@@ -148,7 +148,7 @@ export class ContractRequestHandler implements IContractRequestHandler {
         {
           skipRevocation: query.skipClaimRevocationCheck ?? false,
           challenge: opts.challenge,
-          verifierDid
+          verifier: DID.idFromDID(verifierDid).bigInt()
```

Comment:
this is an issue. we must take verifier id from contract address (convert eth address to big int (with bytes swap)

Reply:
fixed